content/en/docs/notebooks/troubleshoot.md

```diff
@@ -1,6 +1,6 @@
 +++
-title = "Troubleshooting Guide"
-description = "Fixing common problems in Kubeflow notebooks"
+title = "Troubleshooting guide for Jupyter notebooks on Kubeflow"
```

Comment:
I think this should stay short, especially because given context (its under the "jupyter notebook tab) its obviously about the notebooks.

I would say keep this as "Troubleshooting Guide"

Reply:
I totally agree since it is already in the "Jupyter Notebooks" section.
WDYT, @8bitmp3 @RFMVasconcelos?

Reply:
@kelvins if you make this change and squash your commits, I will merge this right now

Reply:
The 1st reason for suggesting a longer title is because "Troubleshooting guide" may be too generic, since readers scan pages and there's a lot of Kubeflow stuff on our site. The 2nd one is to help the search engines. Having both "troubleshooting" and "Fixing common problems" in the title and the description helps with keyword search, even if the site uses Google, which is probably powered by BERT (NLP is hard, anyway).

How about we get rid of the obvious part as follows:

```suggestion
title = "Troubleshooting Guide for Jupyter Notebooks"
```

(not using sentence case here, since our titles, apparently, don't follow the Google style guide across the site :) )

Anyway, you don't have to OK my suggestions :) I'm just trying to help

Reply:
@kelvins I agree with @8bitmp3